docs/tool/GUI.ModsManager/index.md

```diff
@@ -0,0 +1,378 @@
+        source:
+          - name: title/title1.png
+            highdef: title/title1_hd.png
+            index: 1
+```
+
+* `kh2msg` (KH2) - Replaces text found within a kh2 messages file. Uses a yaml file as an source.
+
+Asset Example
+
+```
+  - name: msg/jp/sys.bar
+    method: binarc
+    source:
+      - name: sys
+        type: list
+        method: kh2msg
+        source:
+          - name: sys.yml
+            language: jp
+```
+
+Yaml Source Example
+
+```
+- id: 0x432e
+  en: OpenKH is awesome!
+  it: OpenKH è incredibile!
+  sp: ¡OpenKH es increíble!
+  gr: OpenKH ist großartig!
+  fr: OpenKH est incroyable!
+  jp: OPENKHすばらしい!
+```
+
+* `areadatascript` (KH2) - Modifies a series programs found within a KH2 Spawnscript subfile (located within ard files), using the text format created by OpenKh.Command.SpawnScript. You can only provide a subset of the programs found within the spawnscript, the others will be taken from the original file.
+
+Asset Example
+```
+- method: binarc
+  name: ard/hb34.ard
+  source:
+  - method: areadatascript
+    name: evt
+    source:
+    - name: files/hb34/program-87
+    - name: files/hb34/program-7c
+    - name: files/hb34/program-7d
+    - name: files/hb34/program-86
+    type: AreaDataScript
+```
+
+Text Source Example
+```
+Program 0x7C
+Party DEFAULT
+Bgm Default Default
+AreaSettings 0 -1
+	SetJump Type 2 World HB Area 0 Entrance 0 LocalSet 151 FadeType 1
+	SetPartyMenu 0
+```
+
+* `areadataspawn` (KH2) - Modifies a KH2 Spawnpoint subfile (located within ard files), using an yaml file created using OpenKh.Command.SpawnScript.
+
+Asset Example
+
+```
+- method: binarc
+  name: ard/wi03.ard
+  source:
+  - method: spawnpoint
+    name: b_40
+    source:
+    - name: files/b_40.yml
+    type: AreaDataSpawn
+```
+
+* `listpatch` (KH2) - Can modify the following different types of list binaries found within KH2.
+ * `trsr`
+ * `item`
+ * `fmlv`
+ * `lvup`
+ * `bons`
+ * `objentry`
+
+Asset Example
+```
+- name: 00battle.bin
+  method: binarc
+  source:
+  - name: fmlv
+    method: listpatch
+    type: List
+    source:
+      - name: FmlvList.yml
+        type: fmlv
+```
+
+`trsr` Source Example
+```
+2:
+  ItemId: 347
+```
+
+`item` Source Example
+```
+Stats:
+- Ability: 412
+  AbilityPoints: 0
+  Attack: 0
+  DarkResistance: 100
+  Defense: 0
+  FireResistance: 100
+  GeneralResistance: 100
+  IceResistance: 100
+  Id: 116
+  LightningResistance: 100
+  Magic: 7
+  Unknown: 0
+  Unknown08: 100
+  Unknown0d: 100
+Items:
+- Id: 1
+  Type: Consumable
+  Flag0: 0
+  Flag1: 40
+  Rank: C
+  StatEntry: 1
+  Name: 33528
+  Description: 33529
+  ShopBuy: 40
+  ShopSell: 10
+  Command: 23
+  Slot: 0
+  Picture: 1
+  Icon1: 9
+  Icon2: 0
+```
+
+`fmlv` Source Example
+```
+Final:
+- Ability: 578
+  Experience: 12
+  FormId: 5
+  FormLevel: 1
+  GrowthAbilityLevel: 1
+```
+
+`lvup` Source Example
+```
+Sora:
+  2:
+    Ap: 0
+    Character: Sora
+    Defense: 0
+    Exp: 100
+    Level: 2
```

Comment:
'Level' doesn't have any effect from what I can see. I need to check with the code if this value is still used as this was a left over from an older version? Because this confused me while making level up randomiser.

Reply:
Just confirmed, the "Level" field doesn't do anything.
```
Sora: #character  
  1: #level
    Ap:
    Defense:
    Exp:
    Magic:
    Padding:
    ShieldAbility:
    StaffAbility:
    Strength:
    SwordAbility:
  2:
    Ap:
    Defense:
    [...]
```
would be the necessary fields.